src/Tribe/Views/V2/Widgets/Compatibility.php

```diff
@@ -80,20 +83,32 @@ public function switch_compatibility() {
 		add_filter( "option_widget_{$this->primary_id_base}", [ $this, 'merge_list_widget_options' ] );
 	}
 
+	public function is_v2_adv_list_widget() {
```

Comment:
:no_entry_sign: Missing doc comment for function is_v2_adv_list_widget()

<sub>hash: 81adc0d9472315ad9e86f7aa31072584</sub>